data/json/mapgen/dairy_farm.json

```diff
@@ -0,0 +1,244 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "dairy_farm_NW" ],
+    "weight": 250,
+    "object": {
+      "rows": [
+        "........................",
+        ".$$$$$$$$$$$$$$$$$$$$$$$",
+        ".$......................",
+        ".$......................",
+        ".$.................... .",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$................... ..",
+        ".$................... ..",
+        ".$................... ..",
+        ".$..................... ",
+        ".$......................",
+        ".$......................",
+        ".$......................"
+      ],
+      "terrain": {
+        " ": "t_grass",
+        "$": "t_chainfence_v",
+        ".": "t_grass"
+      },
+      "place_monster": [
+        { "monster": "mon_cow", "x": 7, "y": 4 },
+        { "monster": "mon_cow", "x": 14, "y": 4 },
+        { "monster": "mon_cow", "x": 19, "y": 6 },
+        { "monster": "mon_cow", "x": 9, "y": 10 },
+        { "monster": "mon_cow", "x": 11, "y": 10 },
+        { "monster": "mon_cow", "x": 17, "y": 11 },
+        { "monster": "mon_cow", "x": 7, "y": 12 },
+        { "monster": "mon_cow", "x": 9, "y": 14 },
+        { "monster": "mon_cow", "x": 13, "y": 16 },
+        { "monster": "mon_cow", "x": 6, "y": 17 },
+        { "monster": "mon_cow", "x": 8, "y": 18 },
+        { "monster": "mon_cow", "x": 13, "y": 18 }
```

Comment:
Don't place each monster one by one, place a group like this:
```
{ "monster": "GROUP_COWS", "x": [ 1, 22 ], "y": [ 1, 22 ], "density": 0.05 }
```
Then define the cow group somewhere on top of the file.